app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -23,6 +23,11 @@ def refresher_options
         options[self.class.ems_type]
       end
 
+      # Rhev specific method
+      def self.get_target(event_hash)
+        self.class.try(:get_target, event_hash)
```

Comment:
I checked how refresh is working and I see that we create new instance of a refresher and call refresh. Here we do not have targets to create an instance.

In the way it is implemented now it is enough to implement  `def self.create_new_target(event)` for specific provider.

Btw based on my experience with `save_inventory` I do not think that it would be possible to persist partial data due to deletes that would occur.